### ISSUE TITLE
[Component Proposal] Date and Time range picker

### ISSUE THREAD

**Issue by daphoenix** [IBM/carbon-addons-iot-react#688]
### Summary

Describe in detail the component you would like developed.
Include what offering/product you plan to utilize this new component in.

The Date and Time range picker for Building Insights for [next release](https://ibm.invisionapp.com/share/KDNZFOF3JBF#/screens/319302295_5-1-UtilRate-Interactive-TimeFilter)

**Additional context**
Add any other context about the component here [e.g. screenshots, code, InVision prototype, Box notes containing details necessary for implementation]

[Invision prototype screen](https://ibm.invisionapp.com/share/KDNZFOF3JBF#/screens/319302295_5-1-UtilRate-Interactive-TimeFilter)
Should be able to :
1. Select a date and time from the calendar widget.
or 
2. Select a start time period from the choice of last 24 hours, 7 days, 30 days, 3 months, 6 months or 12 months.
<img width="613" alt="Screenshot 2019-11-05 at 22 44 33" src="https://user-images.githubusercontent.com/5711817/68248879-f6081d80-001d-11ea-93d3-dd734e1adfa6.png">

### Specific timeline issues / requests
Do you want this work within a specific time period? Is it related to an
upcoming release?
* Required by the end of November for next product release.

_NB: The core contributors will try to work with your timeline, but it's not
guaranteed. The earlier you make a request in advance of a desired delivery
date, the better!_

### You will consider this component complete when...
- [ ] It's available as an export of this library
- [ ] Additional criteria you may have [e.g. Fully responsive, Accessible to WCAG 2.0, etc.]

### Available extra resources
What resources do you have to assist this effort?

Possibly one member of the development team, depending on timelines.
_Carbon Add-ons for Watson IoT is a collaborative system. We encourage teams to build components and submit them for integration._

**Comment by tay1orjones** [comment]
I think this is a dupe of #664 ?

**Comment by daphoenix** [comment]
Yes and closing
